baseapp/baseapp.go

```diff
@@ -210,9 +233,9 @@ func (app *BaseApp) initFromStore(mainKey sdk.StoreKey) error {
 // NewContext returns a new Context with the correct store, the given header, and nil txBytes.
 func (app *BaseApp) NewContext(isCheckTx bool, header abci.Header) sdk.Context {
 	if isCheckTx {
-		return sdk.NewContext(app.checkState.ms, header, true, nil, app.Logger)
+		return sdk.NewContext(app.checkState.ms, header, true, nil, app.Logger, 0)
```

Comment:
can we remove passing the gaslimit here too ? Seems like we're not using it anymore (InfiniteGasMeter ?)